docs/specs/sequences/generated.md

```diff
@@ -120,6 +142,18 @@ device publishes pointset events at a rate of no more than config sample_rate_se
 1. Wait for receive at least 5 pointset events
 1. Check that time period between successive pointset events is between 1 and 5 seconds
 
+## state_make_model (BETA)
+
+device publishes correct make and model information in state messages
+
+1. Test skipped: make and model not defined in metadata
```

Comment:
need a model update!

Reply:
model update done and now referenced in CI testing